### ISSUE TITLE
[Win7][Classic][Fx57]: Show Searchbar Labels doesn't autosize terms list height

### ISSUE THREAD

**Issue by MarkRH** [Aris-t2/CustomCSSforFx#40]
Screenshot easiest to explain.  
Before: 
![image](https://user-images.githubusercontent.com/8035962/32879938-83d33da0-ca71-11e7-81d6-5f14d7201765.png)
After:

![image](https://user-images.githubusercontent.com/8035962/32879930-76b5a0ea-ca71-11e7-8cfe-7486d9092953.png)

Notice the big gap after the second term. Be cool if it autosized as well. I'll see if I can figure something out.

(Gah, content security policy preventing showing the images directly.. ok figured how to upload into post)

**Comment by MarkRH** [comment]
Tried looking at it with the Developer Toolbox but the disable popup hide doesn't seem to work with the Searchbar Dropdown. Playing with the CSS, I know why it has a min-height and height of 180px because otherwise it chops the list off.  Hmm.

**Comment by Aris-t2** [comment]
I've set a min-width for that area, but it causes what is visible on your screnshot.

Edit searchbar_popup_engines_show_labels.css and change the code to this:

```css
#PopupSearchAutoComplete .search-panel-tree {
  min-height: auto !important;
  height: auto !important;
}
```

Will be fixed on next update.

**Comment by MarkRH** [comment]
Hmm.. I put that code in and it looks like this now:

![image](https://user-images.githubusercontent.com/8035962/32889913-586aef16-ca92-11e7-964d-4e170abb3a5c.png)

I'll wait for your update and see what happens then.

**Comment by Aris-t2** [comment]
Maybe your search engine list is too long?

**Comment by MarkRH** [comment]
Looks like with the height set to auto, even adding 1 search engine to the defaults starts breaking it.  The more engines I add, the more it squashes the terms list.  
![image](https://user-images.githubusercontent.com/8035962/32917428-c74c3ab4-cae4-11e7-808c-e9d4c4b4f3b0.png)
Fewer engines:
![image](https://user-images.githubusercontent.com/8035962/32917485-f5df470e-cae4-11e7-964d-de32b9e2c406.png)

Oh well, just gonna stick it back to 180px and put the engines I want back in.

**Comment by Aris-t2** [comment]
I have a strange idea to catch all cases this popup result tree offers.

Replace previous code with this one (it works for me with 8 engines for example)

```css
#PopupSearchAutoComplete .search-panel-tree[height="180"]:not([collapsed="true"]) {
  min-height: 180px !important;
}
#PopupSearchAutoComplete .search-panel-tree[height="162"]:not([collapsed="true"]) {
  min-height: 162px !important;
}
#PopupSearchAutoComplete .search-panel-tree[height="144"]:not([collapsed="true"]) {
  min-height: 144px !important;
}
#PopupSearchAutoComplete .search-panel-tree[height="126"]:not([collapsed="true"]) {
  min-height: 126px !important;
}
#PopupSearchAutoComplete .search-panel-tree[height="108"]:not([collapsed="true"]) {
  min-height: 108px !important;
}
#PopupSearchAutoComplete .search-panel-tree[height="90"]:not([collapsed="true"]) {
  min-height: 90px !important;
}
#PopupSearchAutoComplete .search-panel-tree[height="72"]:not([collapsed="true"]) {
  min-height: 72px !important;
}
#PopupSearchAutoComplete .search-panel-tree[height="54"]:not([collapsed="true"]) {
  min-height: 54px !important;
}
#PopupSearchAutoComplete .search-panel-tree[height="36"]:not([collapsed="true"]) {
  min-height: 36px !important;
}
#PopupSearchAutoComplete .search-panel-tree[height="18"]:not([collapsed="true"]) {
  min-height: 18px !important;
}
```

**Comment by B00ze64** [comment]
You're going to have to catch a lot - I have 14 search engine choices myself. Maybe explain in comments in the CSS how to add additional heights so people can fix it when it breaks. I didn't know we could turn that horrible One-Click thing into something usable (e.g. with labels) - Glad I'm following this repo :-)

**Comment by Aris-t2** [comment]
Using this should not require any additional modifications. Result tree is 180px by default and can only shrink from there. I believe all cases are covered for now.

**Comment by MarkRH** [comment]
Hey, this works with my current 13 search engines. It shortens the list as the entries go away. Cool. :)

**Comment by Aris-t2** [comment]
Good to hear.

Issue can not be closed.

Fix is in 1.3.0 release.
